# Node Name: handover_controller
handover_controller:

  ros__parameters:

    # UR5e Parameters
    payload: 5.0 # kg
    reach: 0.85 # m
    max_speed: 180.0 # deg/s
    stopping_time: 0.150 # s
    stopping_distance: 0.20 # m
    position_uncertainty: 0.03 # mm
